### PR TITLE
Initial setup of Sprinkler

### PR DESCRIPTION
**Pull request by SteveLinden**
## A reference to the issue / Description of it

Related to #5980 

This is the initial attempt to set up PagerDuty on Sprinkler so that it's the same as current settings.

### CODE REVIEW

File: terraform/pagerduty/member-services-integrations.tf [hunk 1]
```diff
@@ -2042,3 +2042,37 @@ resource "pagerduty_event_orchestration_service" "corporate-staff-rostering-prep
     actions {}
   }
 }
+
+resource "pagerduty_service" "sprinkler-development" {
+  name                    = "sptinkler-development"
```
Comment (dms1981) [inline terraform/pagerduty/member-services-integrations.tf:2047]: Misspelling here?

File: terraform/pagerduty/member-services-integrations.tf [hunk 2]
```diff
@@ -2042,3 +2042,37 @@ resource "pagerduty_event_orchestration_service" "corporate-staff-rostering-prep
...
+  description             = "sprinkler-development"
+  auto_resolve_timeout    = 345600
+  acknowledgement_timeout = "null"
+  escalation_policy       = pagerduty_escalation_policy.member_policy.id
+  alert_creation          = "create_alerts_and_incidents"
+}
+
+resource "pagerduty_service_integration" "sprinkler-integration" {
```
Comment (dms1981) [inline terraform/pagerduty/member-services-integrations.tf:2055]: Do we need to include commented-out sections? Probably not.

Comment (SteveLinden) [reply terraform/pagerduty/member-services-integrations.tf:2055]: I'll remove the commented out bits.

**Comment by SteveLinden** [comment]
Amendments made based on the comments. Branch re-deployed

**Review by ep-93 — APPROVED** [review]
Dont forget to take this out after testing if its no longer required

**Pull request merged**